package.json

```diff
@@ -38,6 +38,7 @@
   },
   "devDependencies": {
     "@semantic-release/apm-config": "^8.0.0",
+    "@types/atom": "^1.40.5",
```

Comment:
this package isn't written in typescript is this necessary?

Reply:
Datatips in JavaScript come from TypeScript. JavaScript itself is not smart enough to know what is going on in the code. 

Here, I was checking some of the API of Atom to see where decorate API comes from.

Reply:
Can we remove it? I don't see how it helps.

Reply:
It helps the datatips. I reverted it anyway.